.github/GenerateCategories.java

```diff
@@ -0,0 +1,96 @@
+import java.io.File;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.SortedSet;
+import java.util.TreeMap;
+import java.util.TreeSet;
+
+public class GenerateCategories {
+
+    /*
+     *
+     * Generate a JSON string in the format of:
+     //@formatter:off
+     {
+       "include": [
+         { "category": "foo" },
+         { "category": "bar" }
+       ]
+     }
+     //@formatter:on
+     */
+
+    static final boolean DEBUG = false;
+
+    public static void main(String args[]) throws Exception {
+
+        debug("Starting");
+
+        debug("Inputs are: " + Arrays.toString(args));
+        String prSummary = null;
+        if (args != null && args.length > 0)
+            prSummary = args[0].toUpperCase();
```

Comment:
It looks like we don't actually pass the PR Summary arg to this method.
Or did I miss it somewhere?